### ISSUE TITLE
README.md is missing

### ISSUE THREAD

**Issue by trickeydan** [sourcebots/media-consent#1]
We should add a readme file so that people know how to create the pdf file.

**Comment by antonnikitin97** [comment]
I agree - I think to anyone new coming into the organisation it might not be immediately clear that you need to run 'make' in order to make the pdf. Also for people coming in using Windows they might not know how to do it...
